src/MapPanel.tsx

```diff
@@ -333,7 +336,9 @@ export class MapPanel extends Component<MapPanelProps> {
   }
 
   componentWillUnmount() {
-    this.subscriptionHandle.unsubscribe();
+    if (this.subscriptionHandle) {
+      this.subscriptionHandle.unsubscribe();
+    }
```

Comment:
would `this.subscriptionHandle?.unsubscribe();` cover this case?

Reply:
Replaced with optional chaining invocation.